cloud/edgecontroller/pkg/devicecontroller/config/kube.go

```diff
@@ -0,0 +1,81 @@
+package config
+
+import (
+	"time"
+
+	"github.com/kubeedge/beehive/pkg/common/config"
+	"github.com/kubeedge/beehive/pkg/common/log"
+	"github.com/kubeedge/kubeedge/cloud/edgecontroller/pkg/devicecontroller/constants"
+)
+
+// KubeMaster is the url of edge master(kube api server)
+var KubeMaster string
+
+// KubeConfig is the config used connect to edge master
+var KubeConfig string
+
+// KubeNamespace is the namespace to watch(default is NamespaceAll)
+var KubeNamespace string
+
+// KubeContentType is the content type communicate with edge master(default is "application/vnd.kubernetes.protobuf")
+var KubeContentType string
+
+// KubeQPS is the QPS communicate with edge master(default is 1024)
+var KubeQPS float32
+
+// KubeBurst default is 10
+var KubeBurst int
+
+// KubeUpdateNodeFrequency is the time duration for update node status(default is 20s)
+var KubeUpdateNodeFrequency time.Duration
```

Comment:
Is it necessary?

Reply:
It was an extra field, i have deleted. Thanks